frontend/src/metabase/query_builder/components/SidebarContent.jsx

```diff
@@ -10,6 +10,7 @@ export default function SidebarContent({
   className,
   icon,
   title,
+  titleOverride,
```

Comment:
I don't understand why `SidebarContent` needs a `titleOverride` prop. Can't you just do something like

```
<SidebarContent
  title={sidebarTitleOverride  ? sidebarTitleOverride.title : "normal title"}
  onBack={sidebarTitleOverride ? sidebarTitleOverride.onBack : whatever}'
>
```

or even 

```
<SidebarContent
  title="normal title"
  onBack={whatever}
  {...(sidebarTitleOverride || {})}
>
```

Maybe call it `sidebarPropsOverride` too.

Reply:
much better idea! I'll change it to `sidebarPropsOverride`

Reply:
updated